### PR TITLE
Update input configuration.

### PR DESCRIPTION
**Pull request by josiah-tesfu**
Updated the input configuration. Configuration is initially set through the editor. Once the node is deployed, configuration can be set dynamically from input. If any parameters from the input message are empty, the previously set configurations are used.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,17 +15,30 @@ npm install @edgepi-cloud/node-red-edgepi-digital-out
...
+- **payload** *string*<br>
+The D-OUT state.
+- **channel** *number*<br>
+
+Example input:
+```
+msg {
+    "payload": "LOW",
+    "channel": 2
+}
```
Comment (farzadpanahi) [inline README.md:32]: provide possible options for state value.

**Review by farzadpanahi — CHANGES_REQUESTED** [review]

**Review by naomiven — DISMISSED** [review]

File: edgepi-digital-out.js [hunk 2]
```diff
@@ -1,51 +1,64 @@
...
 
-    if (dout){
-      console.debug("Digital Output node initialized on:", transport);
-      node.status({fill:"green", shape:"ring", text:"d-out initialized"});
-    }
-    else{
-      node.status({fill:"red", shape:"ring", text:"d-out failed to initialize"})
-      throw new Error("Digital output failed to initialize.")
-    }
+    async function initializeNode(config) {
```
Comment (jimmy121192) [inline edgepi-digital-out.js:31]: `async` is not being used.

**Review by jimmy121192 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (josiah-tesfu) [reply README.md:32]: Fixed.

File: edgepi-digital-out.js (hunk 2 above)
Comment (josiah-tesfu) [reply edgepi-digital-out.js:31]: `await` call on line 17.

**Comment by jimmy121192** [comment]
Approve for now, but i will test it and update the js code ( for `async` call ).

**Review by jimmy121192 — APPROVED** [review]

**Review by farzadpanahi — APPROVED** [review]

**Pull request merged**